trunk/SUAVE/Methods/Aerodynamics/OpenVSP_Wave_Drag/wave_drag_lift.py

```diff
@@ -0,0 +1,67 @@
+# wave_drag_lift.py
+# 
+# Created:  Jun 2014, T. Macdonald
+# Modified: May 2017, T. Macdonald
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+import numpy as np
+from SUAVE.Analyses import Results
+
+# ----------------------------------------------------------------------
+#   Wave Drag Lift
+# ----------------------------------------------------------------------
+
+def wave_drag_lift(conditions,configuration,wing):
+    """ SUAVE.Methods.wave_drag_lift(conditions,configuration,wing)
+        computes the wave drag due to lift 
+        Based on http://adg.stanford.edu/aa241/drag/ssdragcalc.html
+        
+        Inputs:
+        - SUave wing
+        - Sref - wing reference area
+        - Mc - mach number
+        - CL - coefficient of lift
+        - total_length - length of the wing root
+        Outputs:
+        - CD due to wave drag from the wing
```

Comment:
I removed them for now. I'll add updated ones when we go through everything for doxygen